### ISSUE TITLE
section.title in Composition profile should be 0..1

### ISSUE THREAD

**Issue by adamzkover** [HL7/emedicinal-product-info#105]
Some package leaflets have unidentified sections between known subsections. These text blocks do not always have titles.
Consider making the title optional or mandatory only on the first level.

**Comment by cander2** [comment]
Guess it makes sense since the code is optional as well

**Comment by adamzkover** [comment]
The [May 2024 Connectathon Track Report](https://docs.google.com/document/d/1hqoKK764Kkx2YF7WrkLU4CmHUZgh-dyMsiCaJvQjXFY/edit) includes this statement:
"Team agrees Composition.section.title should be 1..1 since we should not allow sections without a title."
This may or may not contradict what we said so far.

I agree that the main, first-level section should have a title. This is what the path "Composition.section.title" suggests.

We should, however, not generally require all sections to have a title. There is sometimes text between identified subsections in real-world package leaflets, so we need uncoded sections. These text blocks often do not have titles either.

**Comment by cander2** [comment]
Hi Adam,
Can you share an example of a label where the section has no title?

FDA's terminology has what's called 42229-5 | SPL UNCLASSIFIED SECTION. This allows manufacturers to add custom headings that are still encoded but they can make up any title they like. Recommend EMA do something similar.

**Comment by adamzkover** [comment]
There are fewer examples than I thought. Going through the sections coded "unspecified", most do have a title. Some that do not could probably be merged into previous sections. For example, in https://www.felleskatalogen.no/medisin/pasienter/pas-ramipril-actavis-actavis-563214 should "Hvis barn har fått i seg legemiddel ved uhell kontakt lege, sykehus eller Giftinformasjonen (tlf. 22 59 13 00)." ("if children accidentally take X") be a part of section "if you take more X than you should"? We have them coded separately now, which leaves this text without a proper title.

The examples below are the ones closest to not having a title:

In https://www.felleskatalogen.no/medisin/pasienter/pil-inuxair-chiesi-farmaceutici-560226, section 3, the last identified subsection is "Dersom du avbryter behandling med Inuxair" - SPOR 200000029909, "If you stop &lt;taking&gt; &lt;using&gt; X". Before section 4 - SPOR 200000029910, "4. Possible side effects", there are several smaller sections. We have them coded together in an "unspecified" section, and collectively they do not have a single title. Maybe these should be coded in separate sections, and then "Dersom pusten din blir verre:", "Dersom astmaen blir verre:", etc. would be titles.

In https://www.felleskatalogen.no/medisin/pasienter/pil-fiasp-fiasp-flextouch-fiasp-penfill-fiasp-pumpcart-novo-nordisk-649184, "Tre enkle tiltak som kan hjelpe for å unngå lavt eller høyt blodsukker er:" is between SPOR sections 200000029908 "If you forget to &lt;take&gt; &lt;use&gt; X" and 200000029909 "If you stop &lt;taking&gt; &lt;using&gt; X". Here I am unsure if "Tre enkle tiltak som kan hjelpe for å unngå lavt eller høyt blodsukker er:" is a title, or rather part of the text.

In https://www.felleskatalogen.no/medisin/pasienter/pas-buventol-easyhaler-orion-547177, 
"Inhalatorens bruksanvisning er bakerst i pakningsvedlegget." is after SPOR section 200000029907 "If you &lt;take&gt; &lt;use&gt; more X than you should". This text is in a separate section, coded "unspecified", but the text itself doesn't seem to fit as title.

**Comment by cander2** [comment]
@adamzkover,
Regarding the examples, why would they not have titles? For instance, the first example, the title would be "if children accidentally take X"

In each example there would be the official EMA heading and then if EMA added a code for "UNCLASSIFIED SECTION" then we all could add whatever sub-sections beneath the official heading and title it whatever we want. 

if you take more X than you should <- EMA SPOR heading
- if you take more X than you should <- coded with "UNCLASSIFIED SECTION" and then industry can add whatever title they want

**Comment by adamzkover** [comment]
We have a good candidate for a subsection title in the first example, that is true. The other examples are not that obvious. Example 4 with Buventol has only a single sentence in that unspecified subsection, and that text doesn't sound like a title. Most unspecified subsections will probably have a title. It would be good however to have the possibility to encode cases where there is no text that could be interpreted as a title. If not for authoring new ePILs in the future, then at least for encoding existing ones to FHIR.

**Comment by cander2** [comment]
Pushed the merge earlier. Does that make it optional now?

**Comment by adamzkover** [comment]
Yes, section.title is optional on http://build.fhir.org/ig/HL7/emedicinal-product-info/StructureDefinition-Composition-uv-epi.html now.
